api/v1alpha1/driver_types.go

```diff
@@ -71,13 +71,13 @@ type PodCommonSpec struct {
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
 }
 
-type PluginResourcesSpec struct {
+type NodePluginResourcesSpec struct {
```

Comment:
As agreed in the last sync meeting, changes to the design doc, to align with the codebase, will follow the completion of the merge of the initial reconcile code. This will guarantee that requirements coming from technical changes are settled.
The underlying idea is to let the APIs settle before we rush to update the design doc